specs/packagemanager/PackageManagement.md

```diff
@@ -667,6 +715,15 @@ namespace Microsoft.Windows.Management.Deployment
         UsePreference,
     };
 
+    /// Defines the stub behavior for an app package that is being added or staged.
+    [contract(PackageDeploymentContract, 1)]
```

Comment:
Have we already shipped the v1 of this contract?

Reply:
Yes - 1.5.0 just shipped yesterday (was a surprise).

RECOMMEND: Change contract to v2 for the new bits.